word_scripts/wordfind_site.py

```diff
@@ -0,0 +1,26 @@
+import requests
+from bs4 import BeautifulSoup
+
+def wordFindSite ():
+    wordFind_dict = {}
+    for word_length in range(3,12):
+        try:
+            req = requests.get("http://www.wordfind.com/{0}-letter-words/".format(word_length))
```

Comment:
If `requests.get` fails we shouldn't run all the other code below so you probably want to move the stuff below into the try